copilot/kubernetes-platform-api.py

```
@@ -0,0 +1,54 @@
+    return client.CoreV1Api()
+
+class DeploymentRequest(BaseModel):
+    name: str
+    image: str
+    replicas: int = 3
+    namespace: str = "default"
+
+@app.post("/deployments/")
+async def create_deployment(
+    deployment: DeploymentRequest, 
+    k8s_api: client.CoreV1Api = Depends(get_k8s_client)
```

Ruff (0.8.2)

Unused function argument: `k8s_api`

(ARG001)